manifests/cf-rabbitmq-server-only-template.yml

```diff
@@ -46,6 +46,8 @@ properties:
     - rabbitmq_management
     - rabbitmq_mqtt
     - rabbitmq_stomp
+    policies:
+      HA: '''^(?!amq\.).*'' ''{"ha-mode": "all"}'''
```

Comment:
Agree, I just wanted to show how to use new feature, but did not found a better place to put an example.
Would you like to comment it out or completely remove?